presto-kafka/src/main/java/io/prestosql/plugin/kafka/KafkaConfig.java

```diff
@@ -44,6 +44,13 @@
     private File tableDescriptionDir = new File("etc/kafka/");
     private boolean hideInternalColumns = true;
     private int messagesPerSplit = 100_000;
+    private String securityProtocol;
```

Comment:
Security protocol should be an enum, so we could fail if it is set to incorrect value.

Reply:
done